styleguide/source/_patterns/03-templates/02-ec-home/01-ec-home-three-up/ec-home-three-up.twig

```diff
@@ -0,0 +1,50 @@
+{% include 'organisms-ribbon' %}
+{% include 'molecules-alert-info' %}
+
+<div class="ec">
+    <section class="ec_search-bar">
+        {% include '02-organisms/05-ec/01-ec-search-bar/ec-search-bar.twig' %}
+    </section>
+
+    {# hero #}
+    <section class="ec_hero layout layout-one-up">
+        {% include 'molecules-hero-banner' with {'heroContent': heroContent} %}
+    </section>
+
+    {# card section #}
+    <section class="ec_cards-landscape">
+        {% include 'organisms-card-row-ec-three-up' with {'cardType': 'landscape'} %}
+        {#{% include 'organisms-card-row-ec-four-up' with {'cardType': 'landscape'} %}#}
+        {#{% include 'organisms-card-row-ec-five-up-left' with {'cardType': 'landscape'} %}#}
```

Comment:
Please remove commented out lines of code and refactor `with {'cardType': 'landscape'}` according to notes in the card-row files above.